sandbox-backend/src/main/resources/db/migration/V0000033__addFavoritesToEmployee.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE employee_discount(
+    id         INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    discountId INT NOT NULL,
+    employeeId INT NOT NULL,
+    FOREIGN KEY (discountId) REFERENCES discount (id),
+    FOREIGN KEY (employeeId) REFERENCES employee (id)
+);
+
+INSERT INTO employee_discount (employeeId, discountId)
```

Comment:
don't need defaults. Database will be cleaned up before demo, hardcoded ID's might cause issues

Reply:
okay